cmd/minikube/cmd/start.go

```diff
@@ -908,3 +913,14 @@ func extractVMDriverVersion(s string) string {
 	v := strings.TrimSpace(matches[1])
 	return strings.TrimPrefix(v, version.VersionPrefix)
 }
+
+func validateOSSupportVMDriver(os, vmDriver string) bool {
+	switch vmDriver {
+	case constants.DriverNone:
+		if os == constants.Windows || os == constants.Darwin {
```

Comment:
You should be looking for Linux here. For instance, if FreeBSD is added - it still won't support running